packages/picasso.js/package.json

```
@@ -27,6 +27,7 @@
   "module": "dist/picasso.esm.js",
   "scripts": {
     "build": "cross-env NODE_ENV=production rollup --config ../../rollup.config.js",
+    "build:dev": "cross-env rollup --config ../../rollup.config.js",
```

```suggestion
    "build": "rollup --config ../../rollup.config.js",
```
I would suggest removing NODE_ENV=production here. As i think it is enough to have in the top level package.json
(same for the package.json in hammer and q plugins)

---

Good point! Updated!